test/dependency-graph.js

```diff
@@ -402,37 +398,58 @@ describe.only("dependency-graph tests", () => {
   describe("checkValidJSON function", () => {
     it("should pass if checkValidJSON does not throw an error", () => {
```

Comment:
we write what the function should / shouldn't do...instead of when the test should / shouldn't pass.
So change this to - 
`should throw error if parseAssign is assigned invalid JSON value`